package.json

```diff
@@ -35,6 +35,7 @@
     "webpack-cli": "^3.1.2"
   },
   "dependencies": {
-    "@types/webpack": "^4.4.19"
+    "@types/webpack": "^4.4.19",
+    "loader-utils": "^1.2.3"
```

Comment:
This is already included in #74 but adding this here to pin the version so that `[contenthash]` is available when testing.